core/client/src/main/java/alluxio/client/block/BufferedBlockInStream.java

```diff
@@ -102,7 +102,7 @@ public int read(byte[] b, int off, int len) throws IOException {
     checkIfClosed();
     Preconditions.checkArgument(b != null, PreconditionMessage.ERR_READ_BUFFER_NULL);
     Preconditions.checkArgument(off >= 0 && len >= 0 && len + off <= b.length,
-        PreconditionMessage.ERR_BUFFER_STATE.format(b.length, off, len));
+        PreconditionMessage.ERR_BUFFER_STATE.toString(), b.length, off, len);
```

Comment:
not sure this works but my impression is guava `Preconditions` only takes `"%s"` rather than `"{0}"`?
please double check